react/week2/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Counter from "./Counter";
+import ToDoList from "./ToDoList";
+
+class App extends React.Component {
+  
+  render() {
+    return (
+      <div>
```

Comment:
If you want to go above and beyond, you should read about [Fragments](https://reactjs.org/docs/fragments.html) and see if you could take advantage of them here.

Reply:
Thanks for the feedback 👍